downstream/modules/automation-hub/proc-adding-an-execution-environment.adoc

```diff
@@ -4,7 +4,7 @@
 = Adding an execution environment
 
 .Procedure
-. Click *Execution Environment* in the main menu.
+. Click btn:[Execution Environment] in the main menu.
```

Comment:
I wonder if this should be Navigate to menu:Execution Environments[]. like in the above procedure.